app/assets/javascripts/discourse/app/components/d-editor.gjs

```diff
@@ -715,45 +716,40 @@ export default class DEditor extends Component {
             {{if this.disabled 'disabled'}}
             {{if this.isEditorFocused 'in-focus'}}"
         >
-          <div class="d-editor-button-bar" role="toolbar">
-            {{#if this.siteSettings.rich_editor}}
-              <ToggleSwitch
+
+          {{#if this.replacedToolbarInstance}}
+            <div class="d-editor-button-bar --replaced-toolbar" role="toolbar">
+              <DButton
+                @action={{fn this.replaceToolbar null}}
```

Comment:
This is not strictly necessary but I would introduce a method called `resetToolbar`or something like it. Just to make it clearer how it should be done.